dev/ssl-stores/client-test-keypair.pem

```diff
@@ -0,0 +1,89 @@
+Bag Attributes
```

Comment:
Just a note that, if/when we start running these tests against prod, we'll need to stop using these static certs and instead use something more secure.

Reply:
Ack.